rele/client.py

```diff
@@ -22,14 +23,21 @@ def __init__(self, gc_project_id, credentials):
         else:
             self._client = pubsub_v1.SubscriberClient(credentials=credentials)
 
-    def create_subscription(self, subscription, topic):
+    def get_default_ack_deadline(self):
+        return int(os.environ.get('DEFAULT_ACK_DEADLINE', self.DEFAULT_ACK_DEADLINE))
+
+    def create_subscription(self, subscription, topic, ack_deadline_seconds=None):
```

Comment:
What would be the use case for passing the `ack_deadline_seconds` into the `create_subscription` function if it can be set via envvar?

Reply:
As far as we know, one subscriber instance is normally created to register several subscriptions into PubSub. Given this scenario, we thought there could be cases in which a single ack deadline couldn't be suitable to all the subscriptions inside a project. For those cases, the `ack_deadline_seconds` argument passed to `create_subscription` would be handy.

Reply:
If this is a feature we want to have we should add the `ack_deadline_seconds` argument to the `sub` decorator.  I suggest to break that feature into a separate issue, and keep this one using the env var to set the ack_deadline.

Reply:
I agree. I created #54 to describe that feature as a separate one. We'll stick to the implementation of the global setting in this PR.